packages/commonwealth/server/controllers/server_communities_methods/update_community_id.ts

```diff
@@ -30,6 +30,25 @@ export async function __updateCommunityId(
   if (!originalCommunity) {
     throw new AppError('Community to rename not found!');
   }
+
```

Comment:
small detail, but we do a single findOne with redirect:community_id OR id:new_community_id and format the error msg according to the result